rules/library/resources.bzl

```diff
@@ -16,16 +16,22 @@ resources_filegroup = rule(
     implementation = _resources_filegroup_impl,
     attrs = {
         "srcs": attr.label_list(allow_files = True, allow_empty = True),
+        "extensions_to_filter": attr.string_list(mandatory = True, allow_empty = True),
     },
     doc = """Wraps a set of srcs for use as `data` in an `objc_library` or `swift_library`,
 or `resources` in an `apple_resource_bundle`.
 """,
 )
 
-def wrap_resources_in_filegroup(name, srcs, **kwargs):
+def wrap_resources_in_filegroup(name, srcs, extensions_to_filter = [], **kwargs):
+    extensions_to_filter = list(extensions_to_filter)
+    for x in ("xcdatamodeld", "xcmappingmodel"):
```

Comment:
Its ok to not provide a way to opt-out of these two being appended, right? Seems fine to me, but just checking.

Reply:
They need to be filtered, otherwise there’s a guaranteed failure in the Apple bundling rules